sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -336,7 +336,7 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
         }.getMessage
 
         assert(
-          message.contains(s"Table $SESSION_CATALOG_NAME.default.ctasJsonTable already exists."),
+          message.contains(s"Table $SESSION_CATALOG_NAME.default.ctasjsontable already exists."),
```

Comment:
It's better to use the actual table name, which is always lower-cased because of Hive Metastore.